### PR TITLE
Feat/greetings

### PR DESCRIPTION
**Pull request by rachaouldbabaali**
### **In this PR, I have done the following**

- [x] Created a new React app called 'hello-react-front-end'
- [x] Initialized your project with Git.
- [x] Created the App component with react-router.
- [x] Created the Greeting component that will display a greeting. Set it up as a route in your App component.
- [x] Created a store, an action and a reducer that will connect to you API endpont to get the random greeting.
- [x] Displayed the random greeting in your Greeting component.

### CODE REVIEW

File: src/components/Greeting.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// app/javascript/components/Greeting.js
+import React, { useEffect } from 'react';
```
Comment (OyePriscilla) [inline src/components/Greeting.jsx]: - [x] Please kindly remove this commented line of code in order to make your codebase clean. 👍

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @rachaouldbabaali 👏 👏, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Implemented React app that displays a random greeting ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Greeting.jsx (hunk 1 above)
Comment (rachaouldbabaali) [reply src/components/Greeting.jsx]: Done, Thank you @OyePriscilla

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @rachaouldbabaali, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] Git flow is used correctly :white_check_mark: 
- [x] Implemented all the changes from the previous reviewer :white_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**